nntoolkit/utils.py

```diff
@@ -216,7 +224,7 @@ def write_model(model, model_file_path):
         filenames.append('b%i.hdf5' % i)
 
     # Create YAML file
-    with open("model.yml", 'w') as f:
+    with future_open("model.yml", 'w') as f:
```

Comment:
This is not good. `open` should be overwritten (as I wrote in the documentation), because this way it works with Python 2.7+ and Python 3.X.